src/amber/cli/templates/app/shard.yml.ecr

```diff
@@ -12,9 +12,6 @@ targets:
   <%= @name %>:
     main: src/<%= @name %>.cr
 
-  amber:
-    main: lib/amber/src/amber/cli.cr
-
```

Comment:
This target is no longer needed!  This was very confusing to see in the shard.yml.  Since amber is now on the docker image, we don't need to build it every time someone wants to use docker.